packages/backend-core/tests/core/utilities/structures/users.ts

```diff
@@ -12,7 +12,7 @@ import { generator } from "./generator"
 import { tenant } from "."
 
 export const newEmail = () => {
-  return `${uuid()}@test.com`
```

Comment:
`test.com` is a private domain owned by who-knows-who. `example.com` is owned by IANA and is specifically reserved for documentation purposes in [RFC2606](https://datatracker.ietf.org/doc/html/rfc2606). I always prefer using these domains, in case anything should happen and we accidentally send emails to these domains.